cms/plugin_base.py

```diff
@@ -115,6 +115,9 @@ class CMSPluginBase(six.with_metaclass(CMSPluginBaseMetaclass, admin.ModelAdmin)
 
     disable_child_plugins = False
 
+    child_plugins_cache = True
```

Comment:
Also above these two add a comment warning the user that disabling these will result in a performance hit depending on how many of these plugins are found on the page.